modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -177,3 +177,21 @@ If missing, SASL is automatically enabled when any of the other `sasl.` keys are
 * `sasl.mechanisms`: (array) List of allowed SASL mechanism names.
 If missing or empty, the system defaults are used.
 * `sasl.allow-insecure`: (boolean) Allow mechanisms that send clear-text passwords. Defaults to false.
+
+== Creating a Secret
+
+You can create a secret in the directory that contains your certificate and key files by using the following command:
+
+----
+$ oc create secret generic -n openshift-logging my-secret \
+ --from-file=tls.key=your_key_file
```

Comment:
Per the style guide, you should wrap any replaceable values in angle brackets (<>) .  So this should be:

--from-file=tls.key=<your_key_file>

(And the following lines as well)

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
Thanks, @JStickler. Sorry I missed that, @libander. The guidelines for doing this are here:  https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values